pypeit/scripts/__init__.py

```diff
@@ -0,0 +1,53 @@
+
+from pypeit.utils import all_subclasses
+from pypeit.scripts import scriptbase
+
+# The import of all the script modules here is what enables the dynamic
+# compiling of all the available scripts below
```

Comment:
i wonder if there's a clever way to use `import *` here to make this more succinct and automatically include new scripts as they get added.

Reply:
Yeah, I had the same thought.  It may be an outdated concern, but I've been warned off using `from whatever import *`.  If it is done, you're supposed to put a `__all__` list in the relevant `__init__` file, which means you need to list all of the relevant modules anyway.  I'm certainly open to suggestions, but I couldn't think of a clever way to do this.  If we do find a clever way, we should put it in `spectrographs/__init__.py`, as well.

Reply:
places like `__init__.py` are cases where it is ~ok to use it. `photutils` is one codebase that i am familiar with that does, but does put `#noqa` on those lines to keep linters from whinging about it.